### DISCUSSION TITLE
sh-rpi Close of Opencpn before Pi is shutdown

### DISCUSSION THREAD

**Discussion by HPN-1956** [hatlabs/discussions#73] [Q&A]
Hi
Is it possible to add a "kill" of opencpn a few seconds before the pi is shut down?

**Comment by mairas** [comment]
You can do a custom poweroff script that does that first and then calls `shutdown -h now`:

https://github.com/hatlabs/SH-RPi-daemon#configuration

**Reply by HPN-1956** [reply]
sorry but I do not have a file called shrpid.conf in the etc folder.
I also have difficulties with the shrpid command. Everything else but shrpid --help get error responses.
example:
pi@openplotter:~ $ shrpid --blackout-time-limit: 10
usage: shrpid [-h] [--i2c-bus I2C_BUS] [--i2c-addr I2C_ADDR]
              [--blackout-time-limit BLACKOUT_TIME_LIMIT]
              [--blackout-voltage-limit BLACKOUT_VOLTAGE_LIMIT]
              [--socket SOCKET] [--socket-group SOCKET_GROUP] [-n]
              [--poweroff POWEROFF] [--conf CONF]
shrpid: error: unrecognized arguments: --blackout-time-limit: 10
